core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -8668,6 +8676,8 @@ public static final class Name {
     public static final String USER_HOSTNAME = "alluxio.user.hostname";
     public static final String USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS =
```

Comment:
is there a reason why get and list are treated separately? i feel we can just use one property? any objection?

Reply:
> is there a reason why get and list are treated separately? i feel we can just use one property? any objection?
updated